domains/gh0stp4wz.json

```
@@ -5,7 +5,23 @@
   },
   "records": {
     "CNAME": "GH0STP4WZ.github.io",
-    "TXT": ["zoho-verification=zb43811690.zmverify.zoho.eu"]
+    "TXT": [
+     "zoho-verification=zb43811690.zmverify.zoho.eu",
+     "v=spf1 include:zohomail.com ~all"
+   ],
+      "MX": [
+        {
+          "target": "mx.zoho.eu",
+          "priority": 10
+        },
+        {
+          "target": "mx2.zoho.eu",
+          "priority": 20
+        },
+        {
+          "target": "mx3.zoho.eu",
+          "priority": 50
+        }
```

misconfigure MX records
```suggestion
        }
     ]
```